src/Batch/Batch/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Removed the breaking change message.
```

Comment:
Removed out-of-date upcoming breaking change message for `New/Get-AzBatchCertificate`.